src/main/java/carracing/Race.java

```diff
@@ -47,4 +42,10 @@ public void playGame(ResultView resultView) {
         }
     }
 
+    public List<Car> getWinners() {
+        return cars.stream()
+                .filter(car -> car.getPosition() == cars.stream().mapToInt(Car::getPosition).max().orElse(0))
+                .collect(Collectors.toList());
```

Comment:
최대 값을 구한 다음 비교해 가독성을 높이면 어떨까요?
```suggestion
    public List<Car> getWinners() {
        int maxPosition = ...;
        return cars.stream()
                .filter(car -> car.samePosition(maxPosition))
                .collect(Collectors.toList());
```